lib/content-services/src/lib/content-node-selector/content-node-selector.component.ts

```diff
@@ -47,21 +47,24 @@ export class ContentNodeSelectorComponent {
 
     onSelect(nodeList: Node[]) {
         this.chosenNode = nodeList;
-        this.updateTitle(nodeList);
+    }
+
+    onSiteChange(siteTitle: string) {
```

Comment:
Would this apply to ADW as well in Copy / Move action? If yes, this is not ok. In ADW in Copy / Move dialog, the dialog title should show the name of the item we're copying, and this title should NOT change.